client/app/api/course/Assessment/Submission/Answer/Programming.js

```diff
@@ -0,0 +1,62 @@
+import BaseAssessmentAPI from '../../Base';
+
+export default class ProgrammingAPI extends BaseAssessmentAPI {
+  // Creates a programming file and updates all existing files for a programming answer
+  createProgrammingFiles(answerId, submissionFields) {
+    const config = {
+      headers: {
+        'Content-Type': 'multipart/form-data',
+        Accept: 'file_types',
+      },
+    };
+
+    const formData = new FormData();
+    ProgrammingAPI.appendFormData(formData, submissionFields);
+
+    return this.getClient().post(`${this._getUrlPrefix()}/${answerId}/programming/create_programming_files`, formData, config);
+  }
+
+  // Deletes a programming file from a programming answer
+  deleteProgrammingFile(answerId, payload) {
+    const config = {
+      headers: {
+        'Content-Type': 'multipart/form-data',
+        Accept: 'file_types',
+      },
+    };
+
+    const formData = new FormData();
+    ProgrammingAPI.appendFormData(formData, payload);
```

Comment:
Actually for delete programming file is there a need for `appendFormData`? Or should we just send the file_id and answer_id as params?

Reply:
Hmm i think i changed the implementation to send just the params already, i'll just remove this.